spec/jobs/async_batch_destroy_job_spec.rb

```diff
@@ -7,12 +7,14 @@
     include_context :init_destination, id: 2, initial_rate: 0.5
     include_context :init_destination, id: 3, initial_rate: 0.7
 
-    subject { described_class.new(model_class, sql_query).perform}
+    subject { described_class.new(model_class, sql_query, who_is).perform }
 
     before :each do
       stub_const('AsyncBatchDestroyJob::BATCH_SIZE', 2)
     end
 
+    let(:who_is) { create :admin_user }
```

Comment:
@stekatk  looks like who_is designed to be an id of admin_user not a object
https://github.com/yeti-switch/yeti-web/pull/195/files#diff-218fbdc3dea6c12aa4b8a084680af7d9R20